frappe/model/document.py

```diff
@@ -1202,8 +1202,8 @@ def round_floats_in(self, doc, fieldnames=None):
 			doc.set(fieldname, flt(doc.get(fieldname), self.precision(fieldname, doc.parentfield)))
 
 	def get_url(self):
-		"""Returns Desk URL for this document. `/app/Form/{doctype}/{name}`"""
-		return "/app/Form/{doctype}/{name}".format(doctype=self.doctype, name=self.name)
+		"""Returns Desk URL for this document. `/app/{doctype}/{name}`"""
+		return f"/app/{self.doctype}/{self.name}"
```

Comment:
@AkshayKumarTripathi 

Thanks for the contribution. 

Your current implementation returns non slugged names of doctypes

![image](https://user-images.githubusercontent.com/28212972/119484299-811d4580-bd73-11eb-80fb-a4e8c3fb1b7d.png)

We have switched to sluggified names
This would be the correct output
app/blog-post/sd

Check the `slug` function in frappe.desk.utils

Reply:
Implemented sluggified names for get_url() please review.
![Screenshot from 2021-05-25 17-48-10](https://user-images.githubusercontent.com/50769001/119519049-b9824b00-bd96-11eb-86d6-bcdf8c2c6b00.png)

Reply:
Don't slug the name just the doctype

Reply:
I did the above-mentioned change.